mono/mini/aot-compiler.c

```diff
@@ -13281,7 +13281,8 @@ mono_compile_assembly (MonoAssembly *ass, guint32 opts, const char *aot_options,
 	acfg->aot_opts.nftnptr_arg_trampolines = 128;
 #endif
 	acfg->aot_opts.nunbox_arbitrary_trampolines = 256;
-	acfg->aot_opts.llvm_path = g_strdup ("");
+	if (!acfg->aot_opts.llvm_path)
+		acfg->aot_opts.llvm_path = g_strdup ("");
```

Comment:
so I assume `$MONO_LLVM_PATH_OPTION` should eventually end up in `acfg->aot_opts.llvm_path`. Where does that happen?

Reply:
In this commit, we put it in the AOT_BUILD_FLAGS. Look at configure.ac.

https://github.com/mono/mono/pull/14607/files#diff-67e997bcfdac55191033d57a16d1408aR6633

Reply:
It is the only way to conditionally provide it to in-tree invocations without including it in the shipped artifacts at all.